crates/api_models/src/refunds.rs

```diff
@@ -163,11 +157,30 @@ pub struct RefundListResponse {
     pub size: usize,
     /// The List of refund response object
     pub data: Vec<RefundResponse>,
+    // List of available filter
+    pub filter: RefundListMetaData,
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct RefundListMetaData {
+    pub connector: Vec<String>,
+    pub currency: Vec<Currency>,
+    pub status: Vec<enums::RefundStatus>,
 }
```

Comment:
Please add documentation comments for this struct and fields, derive `ToSchema` and add any details necessary for the OpenAPI spec.